### ISSUE TITLE
Error on e-mail integrity

### ISSUE THREAD

**Issue by ghost** [laudeon/express-form-handler#2]
Hi ! I found an error when I check a field with "email" type.

```
var formHandler = require('express-form-handler');
var schema = formHandler.create({
  fields: {
    email: { label: 'Email', type: 'email', required: true }
  }
});
```

When I submit the form with nothing in the email field, I got an error message saying "it's required", but if I write for example "dsq", the form is submitted... And in my view, when I list errors, if I submit 4 times, I have 4 errors for the email field.

**Comment by loulafripouille** [comment]
Hi,

I think after fix that and merge your pull request I can bump to a minor version.

Thanks

**Comment by loulafripouille** [comment]
this was fixed by the 1.1.5 tag (which contains also your pull request)
